model_analyzer/config/config.py

```diff
@@ -144,6 +144,8 @@ def _fill_config(self):
                         field_type=ConfigPrimitive(str, required=True),
                         description='Model repository location'))
 
+        objectives_scheme = ConfigObject(
+            schema={'**': ConfigPrimitive(type_=int)})
```

Comment:
Can you be more explicit about the list of parameters allowed here? I think we know all the keys that should be allowed under this key.

Reply:
Also, I think we should be more flexible here and allow the user to both provide a list of objectives + a dictionary. If a list is provided the weights will be equal. You can unify the output results using the `output_mapper` and creating a new class that contains the unified output. This way we won't have to specify a default weight for the output too.

Reply:
If we are explicit, we will have to update the list of keys when a new metric is added, removed, or existing metric name is changed. I will increase the flexibility using `output_mapper`, but do you think a separate class is necessary here?

Reply:
Actually we can be explicit by getting all registered record types from record.py

Reply:
Right, sounds good!